Configuration of defaults.

mcourteaux opened the pull request:
- Button to save current options to the global Tracy Config.
 - red stars indicating which options get saved.
 - Added a bunch of settings I expect people to like having defaults configurable for.
 - Tooltip explaining you can edit the global config file manually as well.

<img width="676" height="720" alt="tracy_options" src="https://github.com/user-attachments/assets/8391b869-f370-4ae2-90c5-a90e898d15b9" />

 (fixed that awkward space after the red star in the tooltip after taking this screenshot)

Fixes #1139

slomp commented:
If I understand correctly, this will save the current settings to the global config file, such that on subsequent runs the settings are preserved. I think the wording "Save **defaults**" is a bit confusing in that case. Maybe a simple "Save **options** ", or "Remember these options" instead?

wolfpld commented:
The red markers look a bit jarring to me on the screenshots. I do not think they need to be constantly visible.

mcourteaux commented:
I changed it such that only when you hover the button, it changes from gray to red.

<img width="470" height="589" alt="image" src="https://github.com/user-attachments/assets/78aebf79-ae7f-4b5a-afa1-044498a4a3c7" />

I do have another issue that appeared now: the UI stopped updating unless I move the mouse. I would be super surprised if that is due to my changes. More likely: I have updated all packages on my system with `dnf upgrade` yesterday...

mcourteaux commented on profiler/src/profiler/TracyView_Options.cpp:
```diff
@@ -824,6 +848,53 @@ void View::DrawOptions()
+    }
+    else
+    {
+        default_markers_active = false;
```
What do you mean? It does work for me. There will be one frame delay between hovering and coloring, but that's fine for an app that runs with a continuous frame loop.
